src/scripts/main.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
 
-document.addEventListener('click', (e) => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.addEventListener('click', (e) => {
+  const wallRect = wall.getBoundingClientRect();
+  const spiderRect = spider.getBoundingClientRect();
+
+  let targetX = e.clientX - wallRect.left - spiderRect.width / 2;
+  let targetY = e.clientY - wallRect.top - spiderRect.height / 2;
```

Comment:
The calculation of `targetX` and `targetY` is correct, but it might be clearer to explicitly mention that these calculations are intended to center the spider element at the click position. Consider adding a comment to clarify this intention.